src/modules/tasks.js

```diff
@@ -49,17 +50,24 @@ class Tasks {
     });
   }
 
+  delete(id) {
+    // const tasks = document.getElementById('to-do-list');
+    // const mainParent = document.getElementById(`${id}_item`)
+    // tasks.removeChild(mainParent);
```

Comment:
- [ ] [OPTIONAL] ⚠️ Please remove these commented codes 🙏 They make **your codebase looks not clear and it is a bad practice**